config/crd/bases/addonmgr.keikoproj.io_addons.yaml

```diff
@@ -665,6 +665,9 @@ spec:
                     type: string
                 type: object
               type: array
+            starttime:
+              format: int64
```

Comment:
starttime is  epoch time in second?

Reply:
This is carry-over from your status change, this is auto-generated.